eZ/Publish/Core/MVC/Symfony/View/ContentView.php

```diff
@@ -39,139 +40,50 @@
  * }
  * </code>
  */
-class ContentView implements ContentViewInterface
+class ContentView extends BaseView implements View, ContentValueView, LocationValueView
 {
-    /**
-     * @var string|\Closure
-     */
-    protected $templateIdentifier;
-
-    /**
-     * @var array
-     */
-    protected $parameters;
-
-    /**
-     * @var array
-     */
-    protected $configHash;
-
-    /**
-     * @param string|\Closure $templateIdentifier Valid path to the template. Can also be a closure.
-     * @param array $parameters Hash of parameters to pass to the template/closure.
-     *
-     * @throws \eZ\Publish\Core\Base\Exceptions\InvalidArgumentType
-     */
-    public function __construct($templateIdentifier = null, array $parameters = array())
-    {
-        if (isset($templateIdentifier) && !is_string($templateIdentifier) && !$templateIdentifier instanceof \Closure) {
-            throw new InvalidArgumentType('templateIdentifier', 'string or \Closure', $templateIdentifier);
-        }
-
-        $this->templateIdentifier = $templateIdentifier;
-        $this->parameters = $parameters;
-    }
-
-    /**
-     * @param array $parameters Hash of parameters to pass to the template/closure
-     */
-    public function setParameters(array $parameters)
-    {
-        $this->parameters = $parameters;
-    }
-
-    /**
-     * Adds a hash of parameters to the existing parameters.
-     *
-     * @param array $parameters
-     */
-    public function addParameters(array $parameters)
-    {
-        $this->parameters += $parameters;
-    }
+    /** @var \eZ\Publish\API\Repository\Values\Content\Content */
+    private $content;
 
-    /**
-     * @return array
-     */
-    public function getParameters()
-    {
-        return $this->parameters;
-    }
+    /** @var \eZ\Publish\API\Repository\Values\Content\Location */
```

Comment:
It is, it returns null when no location was requested (no `locationId` or `location`) params.